app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -103,21 +103,17 @@ def textual_volumes
   end
 
   def textual_group_endpoints
-    return unless @record.connection_configurations.hawkular
+    endpoints = @record.endpoints.where.not(:role => 'default')
+    return if endpoints.nil?
 
-    TextualGroup.new(
-      _("Endpoints"),
+    endpoint_groups = endpoints.map do |e|
       [
-        {
-          :label => _('Hawkular Host Name'),
-          :value => @record.connection_configurations.hawkular.endpoint.hostname
-        },
-        {
-          :label => _('Hawkular API Port'),
-          :value => @record.connection_configurations.hawkular.endpoint.port
-        }
+        {:label => _("#{e.role.capitalize} Host Name"), :value => e.hostname},
+        {:label => _("#{e.role.capitalize} API Port"), :value => e.port}
```

Comment:
`#{e.role.capitalize}` inside a gettext won't work. This needs to be (and other occurences):
```ruby
_("%{role} Host Name") % {:role => e.role.capitalize}
```